pkg/applicationdefinitions/system-applications/cluster-autoscaler-app.yaml

```diff
@@ -1,22 +1,16 @@
-#                Kubermatic Enterprise Read-Only License
-#                       Version 1.0 ("KERO-1.0”)
-#                   Copyright © 2025 Kubermatic GmbH
+# Copyright 2025 The Kubermatic Kubernetes Platform contributors.
 #
-# 1.	You may only view, read and display for studying purposes the source
-#    code of the software licensed under this license, and, to the extent
-#    explicitly provided under this license, the binary code.
-# 2.	Any use of the software which exceeds the foregoing right, including,
-#    without limitation, its execution, compilation, copying, modification
-#    and distribution, is expressly prohibited.
-# 3.	THE SOFTWARE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND,
-#    EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
-#    MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
-#    IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
-#    CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
-#    TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
-#    SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
 #
-# END OF TERMS AND CONDITIONS
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
 
 apiVersion: apps.kubermatic.k8c.io/v1
 kind: ApplicationDefinition
```

Comment:
I would recommend adding this label in the YAML file as well since we already know that this should be always present.

```yaml
labels:
    apps.kubermatic.k8c.io/managed-by: kkp
```